autoscheduler/scraper/management/commands/scrape_courses.py

```diff
@@ -1,6 +1,106 @@
+import asyncio
+import time
+from typing import List
 from django.core.management import base
+from scraper.banner_requests import BannerRequests
+from scraper.models import Course, Instructor, Section, Meeting
+from scraper.models.course import generate_course_id
+from scraper.models.section import generate_meeting_id, generate_meeting_time
+
+def parse_section(courseData,instructor_object):
+    #gets values to create object
+    section_id = int(courseData['id'])
+    section_subject = courseData['subject']
+    course_number = courseData['courseNumber']
+    section_number = int(courseData['sequenceNumber'])
+    section_term_code = courseData['term']
+    section_min_credits = courseData['creditHourLow']
+    section_max_credits = courseData['creditHourHigh']
+    section_max_enrollment = courseData['maximumEnrollment']
+    section_current_enrollment = courseData['enrollment']
+    section_instructor = instructor_object
+    #creates and saves section object
+    section_object = Section(id=section_id, subject=section_subject, course_num=course_number, section_num=section_number, term_code=section_term_code, min_credits=section_min_credits, max_credits=section_max_credits, max_enrollment=section_max_enrollment, current_enrollment=section_current_enrollment, instructor=section_instructor)
+    section_object.save()
+    #calls parse meeting for each meeting time in the section
+    meeting_count=0
+    for i in courseData['meetingsFaculty']:
+        parse_meeting(i,section_object,meeting_count)
+        meeting_count+=1
+
+def parse_meeting(meetingsData,section_object,meeting_count):
+    #gets values to create object
+    meeting_id = generate_meeting_id(str(section_object.section_num), str(meeting_count))
+    meeting_crn = meetingsData['meetingTime']['courseReferenceNumber']
+    meeting_building = meetingsData['meetingTime']['building']
+    #checks which days classes are on and adds them to attend_days
+    meeting_class_days = ['monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday', 'sunday']
+    attend_days = [False,False,False,False,False,False,False]
+    day_string = 'monday'
+    for j in range(0,len(meeting_class_days)):
+        day_string = meeting_class_days[j]
+        if meetingsData['meetingTime'][day_string] == True:
+            attend_days[j] = True
```

Comment:
First, I think this should be abstracted into its own function so we can test it.

Second, I think this can be simplified closer to how you had it previously. Instead of pre-filling` attend_days`, you can do it like:

```python
    meeting_class_days = ['monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday', 'sunday']
    attend_days = []
    for day in meeting_class_days:
        attend_days.append(meetingsData['meetingTime'][day])
```